commands/other/trump.js

```diff
@@ -27,8 +27,8 @@ module.exports = class TrumpCommand extends Command {
             'https://www.whitehouse.gov/wp-content/uploads/2017/11/President-Trump-Official-Portrait-200x200.jpg'
           )
           .setDescription(json.value)
-          .setTimestamp()
-          .setFooter('Powered by tronalddump.io', '');
+          .setTimestamp(json.appeared_at)
+          .setFooter('Powered by tronalddump.io! Quote was posted', ' ');
```

Comment:
you can use the api.tronalddump.io homepage image for the footer image but I'm not sure if its appropriate and can hardly be seen its so small
` .setFooter('Powered by tronalddump.io! Quote was posted', 'https://api.tronalddump.io/img/tronalddump_850x850.png');`

Reply:
yeah it's ok as it is now